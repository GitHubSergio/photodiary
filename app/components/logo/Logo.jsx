// Imports
import React from 'react';
import { Image } from 'react-native';
import logoStyles from './logoStyles';

const Logo = () => {
  return (
    <Image
      style={logoStyles.logoBackground}
      source={require('../../assets/images/logo-resized.png')}
    />
  );
};

export default Logo;
