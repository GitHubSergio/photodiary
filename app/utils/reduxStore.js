export const reduxJSStore = {
  user: {
    isSigning: false,
    userDetails: {
      displayName: 'Masa',
      email: 'masa@mail.com',
      userId: 'YVHTtPKcyzReNwHNv3oyV9ArK8n2',
      status: 200,
      accessToken:
        'eyJhbGciOiJSUzI1NiIsImtpZCI6ImNiOGUwZDk3Mjg2MWIwNGJlN2RjNzVhMWIzYmUzYjIyOWIyNWYyMDUiLCJ0eXAiOiJKV1QifQ.eyJuYW1lIjoiTWFzYSIsImlzcyI6Imh0dHBzOi8vc2VjdXJldG9rZW4uZ29vZ2xlLmNvbS9mYmF1dGgtYzgyYzYiLCJhdWQiOiJmYmF1dGgtYzgyYzYiLCJhdXRoX3RpbWUiOjE1ODI1NDQyMTgsInVzZXJfaWQiOiJZVkhUdFBLY3l6UmVOd0hOdjNveVY5QXJLOG4yIiwic3ViIjoiWVZIVHRQS2N5elJlTndITnYzb3lWOUFySzhuMiIsImlhdCI6MTU4MjU0NDIxOCwiZXhwIjoxNTgyNTQ3ODE4LCJlbWFpbCI6Im1hc2FAbWFpbC5jb20iLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZW1haWwiOlsibWFzYUBtYWlsLmNvbSJdfSwic2lnbl9pbl9wcm92aWRlciI6InBhc3N3b3JkIn19.eobDOIURG9Pn2OLgw-EDuEBIlEEbMrvo6NnQ0fmZ5i4pXDGus1uRjeCCIkCIfeadoxBIY4DeypuTB-kLGwt1srrCV9HmXO0W_yWaTMb8cSF1YZhle3xnzTxYRngUi2unw7mXgjDkzjhmeYNFPVv_Gv5jLx_5EyvEkZ8qsaxiobbgmKySjN-ng01kVvnL4Hho9SUPyWaF-Aiutb-vQzOdoSjQNMVyVyY-yuwBEZpO1yNpH_ywDnBLbyXDtT6jRTohwAomznHRmsY3EkomMsIEQiToRi4tulSqJxBAIBEP60iuA4_7GLls8myEnfTkzUuYCOTB7msvty_iVc-gW-91Vg',
    },
    error: '',
  },
  posts: {
    isFetching: false,
    allPosts: [
      {
        title: 'Bangkok',
        description:
          'Bangkok, Thailand’s capital, is a large city known for ornate shrines and vibrant street life. The boat-filled Chao Phraya River feeds its network of canals, flowing past the Rattanakosin royal district, home to opulent Grand Palace and its sacred Wat Phra Kaew Temple. Nearby is Wat Pho Temple with an enormous reclining Buddha and, on the opposite shore, Wat Arun Temple with its steep steps and Khmer-style spire.',
        image: 'imgUrl',
        likes: 0,
        comments: 0,
        createdAt: '2020-02-20T23:08:31.704Z',
        userId: 'YVHTtPKcyzReNwHNv3oyV9ArK8n2',
        docRef: '0264d0b6-a99f-421f-8d02-f87008888de8',
      },
      {
        title: 'Phuket ',
        description:
          'Phuket City, on Phuket Island, is the capital of Thailand’s Phuket Province. In the Old Town, Thalang Road is lined with colorful 19th-century shophouses and Sino-Portuguese buildings. Built in 1903 by a wealthy tin merchant, Baan Chinpracha mansion has Italian floor tiles, shuttered windows and antique furniture. Set in a 1930s manor, the Thai Hua Museum has exhibits on Phuket’s culture and history.',
        image: 'imgUrl',
        likes: 0,
        comments: 0,
        createdAt: '2020-02-20T23:20:47.603Z',
        userId: 'YVHTtPKcyzReNwHNv3oyV9ArK8n2',
        docRef: '52e6b747-ec3d-4bb3-b445-49cabbc85bca',
      },
      {
        title: 'Koh Samui',
        description:
          "Ko Samui, Thailand’s second largest island, lies in the Gulf of Thailand off the east coast of the Kra Isthmus. It's known for its palm-fringed beaches, coconut groves and dense, mountainous rainforest, plus luxury resorts and posh spas. The landmark 12m-tall golden Big Buddha statue at Wat Phra Yai Temple is located on a tiny island connected to Ko Samui by a causeway.",
        image: 'imgUrl',
        likes: 0,
        comments: 0,
        createdAt: '2020-02-20T23:07:56.308Z',
        userId: 'YVHTtPKcyzReNwHNv3oyV9ArK8n2',
        docRef: '93caf8c8-0bf5-4625-9ffa-bc8b8ed31a56',
      },
    ],
    error: '',
    isCreatingPost: false,
    success: 0,
    postDetails: {
      title: 'Y',
      description: 'Hh',
      image: 'imgUrl',
      likes: 10,
      comments: 10,
      createdAt: '2020-02-22T19:04:16.637Z',
      userId: 'YVHTtPKcyzReNwHNv3oyV9ArK8n2',
      docRef: '8dc4c594-0abb-4fa3-8ab2-a9de6da04ca6',
    },
    isDeleting: false,
  },
};
