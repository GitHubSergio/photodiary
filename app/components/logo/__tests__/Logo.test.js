/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import reduxStore from 'test-utils';
import Logo from '../Logo';

describe('<Logo/>', () => {
  describe('@render', () => {
    it('should render the Logo component', () => {
      const { container } = reduxStore.renderWithRedux(<Logo />);
      expect(container).toMatchSnapshot();
    });
  });
});
