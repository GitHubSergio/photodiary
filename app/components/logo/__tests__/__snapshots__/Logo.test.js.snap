// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo/> @render should render the Logo component 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/assets/images/logo-resized.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#2D2A4A",
        "borderRadius": 100,
        "marginBottom": 10,
      }
    }
  />
</View>
`;
